.golangci.yml

```diff
@@ -47,19 +47,18 @@ linters:
     - goconst
     - gofmt
     - goimports
-    - golint
     - gosimple
```

Comment:
was golint the only removed linter? the others seem like they were just moved around?

what checks are lost by removing golint?

Reply:
Revive is added to replace the deprecation on golint.  According to the docs, revive has more rules than golint.

https://github.com/mgechev/revive

`interfacer` was dropped due to deprecation and doesn't have a replacement.

And then I sorted the list.

Reply:
Ok, I'm good on this. Can you remove the all of the 
```
// nolint:interfacer
```
comments sprinkled around the codebase?